lib/dotcom_web/templates/iframe/root.html.heex

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    {csrf_meta_tag()}
+    <%= if Application.get_env(:dotcom, :dev_server?) do %>
+      <link rel="stylesheet" href={"#{webpack_path()}/css/core.css"} />
+      <link rel="stylesheet" href={"#{webpack_path()}/css/app.css"} />
+      <script defer src={"#{webpack_path()}/core.js"}>
+      </script>
+      <script defer src={"#{webpack_path()}/iframe.js"}>
+      </script>
+    <% else %>
+      <link rel="stylesheet" {static_attributes("/css/core.css")} />
+      <link rel="stylesheet" {static_attributes("/css/app.css")} />
+      <script defer {static_attributes("/js/vendors.js")}>
+      </script>
+      <script defer {static_attributes("/js/iframe.js")}>
```

Comment:
Not sure if this is happening in other browsers, but I do get this in Firefox 

<img width="1120" alt="image" src="https://github.com/user-attachments/assets/1127835b-432a-45e1-b791-7757dd50ba64" />